ui/src/submissions/common/components/RichTextEditor/RichTextEditor.jsx

```diff
@@ -3,14 +3,10 @@ import QuillEditor from 'react-quill';
 
 import 'react-quill/dist/quill.snow.css';
 import './RichTextEditor.scss';
+import EditorToolbar from './EditorToolbar';
 
 const QUILL_MODULES = {
-  toolbar: [
-    ['bold', 'italic', 'underline'],
-    [{ list: 'ordered' }, { list: 'bullet' }],
-    ['link'],
-    ['clean'],
-  ],
+  toolbar: '#toolbar',
```

Comment:
this logic now in 2 files, you could
- just move it to `EditorToolbar` and export it as constant.
- pull parent `div` with `id` from toolbar component to here editor component